--- slackpkgplus.conf.orig	2014-10-02 07:46:34.257856179 -0700
+++ slackpkgplus.conf	2014-10-02 07:47:06.631853965 -0700
@@ -1,3 +1,7 @@
+#### for Slackware64-current
+## cp /etc/slackpkg/slackpkgplus.conf /etc/slackpkg/slackpkgplus.conf.old
+## wget -N https://raw.githubusercontent.com/ryanpcmcquen/linuxTweaks/master/slackware/64/current/slackpkgplus.conf -P /etc/slackpkg/
+
 
 # Enable (on) / Disable (off) slackpkg+
 SLACKPKGPLUS=on
@@ -44,35 +48,55 @@
 #
 # (Use /usr/doc/slackpkg+-*/setupmultilib.sh to setup a multilib configuration)
 
+#PKGS_PRIORITY=( multilib:.* alienbob-current:.* restricted-current:.* )
+#PKGS_PRIORITY=( alienbob-current:.* restricted-current:.* )
+#PKGS_PRIORITY=( multilib:.* ktown:.* alienbob-current:.* restricted-current:.* )
+#PKGS_PRIORITY=( ktown:.* alienbob-current:.* restricted-current:.* )
+
+#PKGS_PRIORITY=( multilib:.* alienbob-current:.* )
+PKGS_PRIORITY=( alienbob-current:.* )
+#PKGS_PRIORITY=( multilib:.* ktown:.* alienbob-current:.* )
+#PKGS_PRIORITY=( ktown:.* alienbob-current:.* )
+
 # List repositories you want to use (defined below)
 # remember to launch 'slackpkg update' if you modify that row.
-REPOPLUS=( slackpkgplus restricted alienbob slacky )
-
+#REPOPLUS=( slackpkgplus restricted alienbob )
+#REPOPLUS=( slackpkgplus alienbob msb )
+REPOPLUS=( slackpkgplus alienbob )
 
 # Define mirrors (uncomment one or more mirror; remember to add it to REPOPLUS)
 # Note: only multilib,alienbob,restricted,slacky repositories support GPG. If you add others
 #       you may need to disable gpg check in slackpkg.conf
 
+
 # Slackware 14.1 - x86_64
 #MIRRORPLUS['multilib']=http://taper.alienbase.nl/mirrors/people/alien/multilib/14.1/
+#MIRRORPLUS['msb']=http://slackware.org.uk/msb/14.1/1.8/x86_64/
+
 MIRRORPLUS['alienbob']=http://taper.alienbase.nl/mirrors/people/alien/sbrepos/14.1/x86_64/
-MIRRORPLUS['restricted']=http://taper.alienbase.nl/mirrors/people/alien/restricted_sbrepos/14.1/x86_64/
-MIRRORPLUS['slacky']=http://repository.slacky.eu/slackware64-14.1/
+#MIRRORPLUS['restricted']=http://taper.alienbase.nl/mirrors/people/alien/restricted_sbrepos/14.1/x86_64/
+#MIRRORPLUS['slacky']=http://repository.slacky.eu/slackware64-14.1/
 
 # use this to keep the slackpkg+ package updated
 MIRRORPLUS['slackpkgplus']=http://slakfinder.org/slackpkg+/
 
+
 # Slackware current - x86_64
 #MIRRORPLUS['multilib']=http://taper.alienbase.nl/mirrors/people/alien/multilib/current/
 #MIRRORPLUS['zerouno']=http://www.z01.eu/repo-slack/slackware64-current/
 
+#MIRRORPLUS['ktown']=http://taper.alienbase.nl/mirrors/alien-kde/current/latest/x86_64/
+
+MIRRORPLUS['alienbob-current']=http://taper.alienbase.nl/mirrors/people/alien/sbrepos/current/x86_64/
+#MIRRORPLUS['restricted-current']=http://taper.alienbase.nl/mirrors/people/alien/restricted_sbrepos/current/x86_64/
+
+
 # Local repository:
 #MIRRORPLUS['alienbob']=file://repositories/alien/sbrepos/14.1/x86/
 #
 # Local packages (you do not need metadata nor 'slackpkg update' command):
 #MIRRORPLUS['myrepo']=dir://repositories/mypackages/
 
-
 #
 # Supported Repositories (see /usr/doc/slackpkg+-* for details):
 #
@@ -90,10 +114,11 @@
 #slackers:      http://www.slackers.it/repository/
 #slacke17:      http://ngc891.blogdns.net/pub/slacke17/slackware{,64,arm}-{14.0,14.1}/
 #studioware:    http://studioware.org/files/packages/slackware{,64}-{13.37,14.0,14.1}/
-#slackonly:	 http://slackonly.com/pub/packages/14.1-x86_64/
+#slackonly:     http://slackonly.com/pub/packages/14.1-x86_64/
 #
 #Does NOT support GPG
 #salixos(*):    http://download.salixos.org/{i486,x86_64}/{13.37,14.0,14.1}/
 #salixext:      http://people.salixos.org/ralvex/repository/x86_64/{14.0,14.1}/
 #rlworkman(*):  http://rlworkman.net/pkgs/{13.37,14.0,14.1}/
 #slackel:       http://www.slackel.gr/repo/{i486,x86_64}/current/
+
